tap_netsuite/netsuite/schemas/object_definition.json

```diff
@@ -7,6 +7,7 @@
     {"displayName": "Status", "name": "entityStatus", "type": "string"},
     {"displayName": "Email", "name": "email", "type": "string"},
     {"displayName": "DateCreated", "name": "dateCreated", "type": "datetime"},
+    {"displayName": "address", "name": "defaultAddress", "type": "string"},
```

Comment:
Is there a reason you set `displayName` to lowercase?

Reply:
nop, Iet me change it